paimon-common/src/main/java/org/apache/paimon/codegen/CodeGenerator.java

```diff
@@ -48,12 +48,7 @@ GeneratedClass<NormalizedKeyComputer> generateNormalizedKeyComputer(
     GeneratedClass<RecordComparator> generateRecordComparator(
             List<DataType> inputTypes, int[] sortFields);
 
-    /**
-     * Generate a {@link RecordEqualiser}.
-     *
-     * @param fieldTypes Both the input row field types and the sort key field types. Records are *
-     *     compared by the first field, then the second field, then the third field and so on. All *
-     *     fields are compared in ascending order.
-     */
-    GeneratedClass<RecordEqualiser> generateRecordEqualiser(List<DataType> fieldTypes);
+    /** Generate a {@link RecordEqualiser} with fields. */
```

Comment:
Whether it is necessary to add comments for params.

Reply:
I think fields (in equal context) is enough to explain.